pytket/extensions/aqt/backends/config.py

```diff
@@ -45,3 +48,75 @@ def set_aqt_config(
     if access_token is not None:
         config.access_token = access_token
     config.update_default_config_file()
+
+
+class AQTAccessToken:
+    """Holds the aqt access token in memory."""
```

Comment:
Do we want to offer an option to store it more permanent?

Reply:
It is still possible to set the token in the pytket config file using `AQTConfig.set_aqt_config`. If this is done, it will be stored there across sessions and retrieved in memory when needed, unless specifically overridden.

Reply:
Perfect